src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/data/NomisPrisonerMergeEvent.kt

```diff
@@ -3,3 +3,14 @@ package uk.gov.justice.digital.hmpps.prisonerfromnomismigration.data
 data class NomisPrisonerMergeEvent(
   val bookingId: Long,
 )
+
+data class PrisonerMergeDomainEvent(
+  val additionalInformation: MergeAdditionalInformationEvent,
+)
+
+data class MergeAdditionalInformationEvent(
+  val nomsNumber: String,
+  val removedNomsNumber: String,
+  val bookingId: Long?,
```

Comment:
these two booking ids do not exist on event right now, but I will change that NOW